package/MDAnalysis/coordinates/TRZ.py

```diff
@@ -222,7 +222,7 @@ def _read_trz_header(self):
             ('force', 'i4'),
             ('p3', 'i4')])
         data = np.fromfile(self.trzfile, dtype=self._headerdtype, count=1)
-        self.title = ''.join(data['title'][0])
+        self.title = ''.join((c.decode('utf-8') for c in data['title'][0])).strip()
```

Comment:
Is there a reason for the second list comprehension?  Wouldn't
`''.join(data['title'][0]).strip().decode('utf-8')`
work?